spec/files/test1.csv

```diff
@@ -0,0 +1,2 @@
+Title,Description,Price,Category
+RewardTitle,RewardDescription,3,Category1

```

Comment:
Would be good to have two or three rows to import

Reply:
Good point! I have added additional rows 👍